modules/nf-core/propr/logratio/meta.yml

```diff
@@ -0,0 +1,70 @@
+name: "propr_logratio"
+description: ALR/CLR transform the data matrix
```

Comment:
Could you expand this a bit? The user needs to know what the module is for, and probably doesn't know what ALR or CLR means.